MPC/modules/IDLBase.pm

```diff
@@ -127,8 +127,8 @@ sub do_cached_parse {
       elsif ($arg =~ /^\-I(.+)/) {
         push(@include, $1);
       }
-      elsif ($arg eq '--default-nested') {
-        $self->{'default_nested'} = 1;
+      elsif ($arg eq '--no-default-nested') {
+        $self->{'default_nested'} = 0;
```

Comment:
Is this being set to 1 somewhere else when `--default-nested` or no arg is present?  Java builds are failing on CI.

Reply:
Up on line 116 we are setting `default_nested` based on if we are being called from the base, which might be the problem. Also we should probably check need both `--default-nested` and `--no-default-nested` though because the current behavior you can use both and the last one is the one that takes effect.